.circleci/config.yml

```diff
@@ -65,12 +65,57 @@ jobs:
                       - project/node_modules
                       - project/documentation/dist
 
-    visual-testing:
+    visual-lightest-medium:
         executor: node8
 
         steps:
             - downstream
-            - run: yarn test:visual:ci
+            - run: yarn test:visual:ci --color=lightest --scale=medium
+            - store_artifacts:
+                  path: test/visual/screenshots-current
+
+    visual-lightest-large:
+        executor: node8
+
+        steps:
+            - downstream
+            - run: yarn test:visual:ci --color=lightest --scale=large
+            - store_artifacts:
+                  path: test/visual/screenshots-current
+
+    visual-light-medium:
+        executor: node8
```

Comment:
unrelated, but how come this is labelled as node8? looks like its actually node 10